src/main/java/com/github/manolo8/darkbot/gui/utils/inspector/ObjectTreeNode.java

```diff
@@ -36,49 +37,50 @@ public boolean isLeaf() {
     public String toString() {
         ObjectInspector.Slot slot = (ObjectInspector.Slot) (this.getUserObject());
         return String.format("%03X %-25s  %-25s %s",
-                slot.offset, (slot.name.length() >= 25) ? slot.name.substring(25) : slot.name, slot.type, strValue);
+                slot.offset, (slot.name.length() >= 25) ? slot.name.substring(25) : slot.name, slot.getType(), strValue);
     }
 
     public void update(DefaultTreeModel model) {
+        long address = this.address.get();
         if (address == 0) {
             return;
         }
 
         ObjectInspector.Slot slot = (ObjectInspector.Slot) (this.getUserObject());
 
-        long oldValue = value;
+        boolean changed = value != address;
```

Comment:
value and address will not be equal for all nodes where addressIsValue is false. the point of the check was to avoid re-computing string values and updating the tree if the value is the same